MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -862,6 +870,12 @@ private async Task<string> ExtractImageInternal(string inputPath, string contain
                 inputArg = "-threads " + threads + " " + inputArg; // HW accel args set a different input thread count, only set if disabled
             }
 
+            if (options.HardwareAccelerationType.Contains("videotoolbox", StringComparison.OrdinalIgnoreCase) && LikelyToBeJellyfinFfmpeg)
+            {
+                // VideoToolbox supports low priority decoding, which is useful for trickplay
+                inputArg = "-hwaccel_flags +low_priority " + inputArg;
```

Comment:
How about we use `./ffmpeg -hide_banner -h full` to query? This only needs to be checked once at startup.
```
  -hwaccel_flags     <flags>      .D.V....... (default ignore_level)
     ignore_level                 .D.V....... ignore level even if the codec level used is unknown or higher than the maximum supported level reported by the hardware driver
     allow_high_depth              .D.V....... allow to output YUV pixel formats with a different chroma sampling than 4:2:0 and/or other than 8 bits per component
     allow_profile_mismatch              .D.V....... attempt to decode anyway if HW accelerated decoder's supported profiles do not exactly match the stream
     unsafe_output                .D.V....... allow potentially unsafe hwaccel frame output that might require special care to process successfully
     low_priority                 .D.V....... attempt to run decode pipeline at a lower priority than is used for realtime decoding
```

Reply:
That's strange. I can find `low_priority` in both linux(gcc) and win(gcc+clang) builds.

These are fixed strings and are usually independent of the host type.
https://github.com/jellyfin/jellyfin-ffmpeg/blob/443d842d1e13723f78d5089280f9f2c589d37fbb/debian/patches/0076-vt-low-priority-keyframe-decoding.patch#L37

Reply:
The whole `-hwaccel_flags` helper text  is missing on macOS

Reply:
It seems like running with `ffmpeg -hwaccel videotoolbox -hwaccel_flags +low_priority -hide_banner -f lavfi -i nullsrc=s=1x1:d=500 -f null -` and check if the command exit with 0 will do it. I prefer this way rather to parse an 1MB long text output.

Reply:
This is a good trick. All it takes is a fake input to let ffmpeg parse whether the input options are correct.

Reply:
Changing to `-hwaccel auto` can make it more versatile, especially if other hwaccels can adopt this flag in the future.

Reply:
I think the `-hwaccel` can be omitted entirely as ffmpeg will not really setup the hardware but only parse the input option.